classes/services/PKPSubmissionService.php

```diff
@@ -0,0 +1,646 @@
+<?php
+
+/**
+ * @file classes/services/PKPSubmissionService.php
```

Comment:
Ah, I didn't realize that was ready for another look. Could you drop a link to the PR into the issue, and when a PR is ready for a second look, request a review or tag me in a comment on the issue?